docs/notebooks/analysis/swept_langmuir/find_floating_potential.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
<p>See my comments for the other notebook on pathlib! <code>join</code>  and <code>abspath</code>  are just darn ugly.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/889/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Ok, I implemented `pathlib`.  Not sure which I prefer, but I might side w/ `pathlib` after I used it more.